service/worker/scheduler/activities.go

```diff
@@ -65,12 +70,19 @@ type (
 	}
 )
 
+const (
+	eventStorageSize = 2 * 1024 * 1024
+	// I do not know the real overhead size, 1024 is just a number
+	recordOverheadSize = 1024
```

Comment:
On first reading, I didn't understand what kind of overhead this is. I now understand that it's the overhead of storing the data, and that we need to deduct it from the storage limit so we don't go over. Could we add a 1-line comment explaining that?